### ISSUE TITLE
port number error

### ISSUE THREAD

**Issue by JoakimFalk** [PHPFusion/PHPFusion#1393]
> I'm running my testsite on a local host using port 8000. But when I try to write this in the relevant field, I get an error and the value is not accepted. This results in errors (see thread_id=38268) when error-administration tries to return to administration without the correct port number.

https://www.php-fusion.co.uk/forum/viewthread.php?thread_id=38272&pid=201462#post_201462

**Comment by JoakimFalk** [comment]
thread_id=38268 , makes no sense to this. Only states that error log can not be cleared, idk if related yet.

**Comment by meangczac** [comment]
I've digged into the issue now.

From my findings select2, and all ajax breaks with wrong port. probably emails too. 

But, site using relative paths and they are not crashing. only site path is critical and is relevant when it comes to crashing the whole system. 

Port : 12345, whilst my localhost is localhost:8000... system running fine.

But, even file inclusion is working with the wrong port. SEO is working as well.

I can even install. No need to touch DB Factory, when I did dbconnect(localhost:8000....) as value, connection all crashes.

I'm waiting for Jan to give me his SQL errors so I can dig further into this issue, normal web hosting environment all is 80, but not that it matters from my findings. So no issues from internet sites for sure.
